guides/common/assembly_building_cloud-images.adoc

```diff
@@ -12,7 +12,7 @@ include::modules/proc_registering-hosts.adoc[leveloffset=+1]
 
 include::modules/proc_installing-the-katello-agent.adoc[leveloffset=+1]
 
-include::modules/proc_installing-the-puppet-agent.adoc[leveloffset=+1]
```

Comment:
This module was a duplicate, so I suggest removing it and using the one below. But it may need some tweaks for the Provisioning Guide appendix C (Satellite).